app/src/main/java/org/xwiki/android/sync/activities/base/BaseActivity.java

```diff
@@ -97,17 +112,30 @@ public void showProgressDialog(String message) {
         progress.show();
     }
 
+    /**
+     * Hide progress dialog if it currently visible
+     */
     public void hideProgressDialog() {
         if (progress != null && progress.isShowing()) {
             progress.dismiss();
             progress = null;
         }
     }
 
+    /**
+     * @return {@link #progress}
+     */
     public ProgressDialog getProgressDialog() {
         return progress;
     }
 
+    /**
+     * Set default behaviour for menu items. Be careful if you want to override this method
+     * @param item The menu item that was selected.
```

Comment:
Better separate general description and parameters by an empty line (much easier to read).

Reply:
fixed